modules/tiles/src/tileset/traversers/tileset-traverser.js

```diff
@@ -177,28 +173,32 @@ export default class TilesetTraverser {
         } else {
           childRefines = child.contentAvailable;
         }
-        refines = refines && childRefines;
+        shouldRefine = shouldRefine && childRefines;
+      }
+
+      if (!shouldRefine) {
```

Comment:
If `shouldRefine` is set to `true`, we can just check `!childRefines` and return, don't need extra `shoulRefine` var ?